supplementary_style_guide/glossary_terms_conventions/general_conventions/x.adoc

```diff
@@ -9,6 +9,43 @@
 
 *See also*: xref:xemacs[XEmacs]
 
+[discrete]
+[[x86_64]]
+==== image:images/yes.png[yes] x86_64 (noun)
+*Description*: A 64-bit version of x86 architecture. Use this format when referring to operating systems and server instances. For example: RHEL, Fedora, CoreOS and other linux distributions. Use in backticks when referring to architecture as a value or parameter.
```

Comment:
```suggestion
*Description*: A 64-bit version of x86 architecture. Use this term when referring to operating systems and server instances, for example RHEL, Fedora, CoreOS and other linux distributions. Use the lowercase format with backticks when referring to architecture as a value or parameter.
```
Can you include a description of when to use the lowercase format without backticks?